source/introduction/what-is-ethereum.rst

```diff
@@ -3,9 +3,9 @@ A next generation blockchain
 =======================================================================
 A blockchain is a distributed computing architecture where every network node executes and records the same transactions, grouped in blocks. Each block contains a mathematical calculation which verifies that it contains all of the transactions and is based on the previous block. If all nodes in the blockchain network have consensus on each block in the chain, then this enables a sort of distributed ledger or replicated database of the transactions that occurred.
 
-Blockchain technology is the technological basis of Bitcoin, first described as a coherent cryptoeconomic system by its mysterius author Satoshi Nakamoto in his white paper "Bitcoin: A Peer-to-Peer Electronic Cash System", published in 2009. While the use of blockchains for more general uses is already discussed in the original paper, it was not until a few years later that blockchain technology emerged as a generic term to designate the cryptoeconomic system composed of a transparent open ledger or database representing a state of a complex system, manipulated via sending transactions to the guardians of the blockchain. The guardians are nodes of a decentralised peer to peer network. In Bitcoin's case the open ledger is simply conceived of as a table of account balances, and transactions are value transfers. Due to the decentralised nature of this peer to peer system, secured by cryptography and driven by algorithmically enforced economic incentives coupled with scarcity and controlled issuence (predictable inflation) coded in the protocol, bitcoin is suitable to serve as sound digital money without central bank or other power behind it. This allows trustless finance between individuals.
+Blockchain technology is the technological basis of Bitcoin, first described as a coherent cryptoeconomic system by its mysterious author Satoshi Nakamoto in his white paper "Bitcoin: A Peer-to-Peer Electronic Cash System", published in 2009. While the use of blockchains for more general uses is already discussed in the original paper, it was not until a few years later that blockchain technology emerged as a generic term to designate the cryptoeconomic system composed of a transparent open ledger or database representing a state of a complex system, updated via sending transactions to the guardians of the blockchain. The guardians are nodes of a decentralised peer to peer network. In Bitcoin's case the open ledger is simply conceived of as a table of account balances, and transactions are value transfers. Due to the decentralised nature of this peer to peer system, secured by cryptography and driven by algorithmically enforced economic incentives coupled with scarcity and controlled issuance (predictable inflation) coded in the protocol, bitcoin is suitable to serve as sound digital money without central bank or other power behind it. This allows trustless finance between individuals.
```

Comment:
All of this is great information, yet exceedingly complex as an intro.
 
I wrote the first draft of this page and I had assumed that it should be readable by a newcomer while still being technically accurate for those who have already studied Bitcoin and blockchain.

 Just my thoughts.